packages/client/src/pages/DashboardPage/RunPage/TracingComponent/TracePanel/TraceTree/index.tsx

```diff
@@ -1,88 +1,74 @@
+        | string
+        | undefined;
+    const format_target = (agentscope?.format as Record<string, unknown>)
+        ?.target as string | undefined;
 
-const SpanNodeTitle = ({
-    name,
-    startTimeUnixNano,
-    latencyNs,
-    attributes,
-}: SpanNodeTitleProps) => {
-    const operationName = attributes.gen_ai?.operation?.name as string;
-    const agent_name = (attributes.gen_ai?.agent?.name as string) || undefined;
-    const model_name =
-        (attributes.gen_ai?.request?.model as string) || undefined;
-    const tool_name = (attributes.gen_ai?.tool?.name as string) || undefined;
-    const format_target =
-        (attributes.agentscope?.format?.target as string) || undefined;
-
-    let displayKind: string;
     if (operationName === 'invoke_agent' && agent_name) {
-        displayKind = operationName + ': ' + String(agent_name);
+        return operationName + ': ' + String(agent_name);
     } else if (operationName === 'execute_tool' && tool_name) {
-        displayKind = operationName + ': ' + String(tool_name);
+        return operationName + ': ' + String(tool_name);
     } else if (
         (operationName === 'chat' ||
             operationName === 'chat_model' ||
             operationName === 'embeddings') &&
         model_name
     ) {
-        displayKind = operationName + ': ' + String(model_name);
+        return operationName + ': ' + String(model_name);
     } else if (operationName === 'format' && format_target) {
-        displayKind = operationName + ': ' + String(format_target);
+        return operationName + ': ' + String(format_target);
     } else if (operationName) {
-        displayKind = operationName;
-    } else {
-        displayKind = 'Unknown';
+        return operationName;
     }
-    return (
-        <div className="flex flex-col w-full py-1 rounded-md">
-            <div className="flex justify-between">
-                <div className="font-[500] truncate break-all max-w-fit">
-                    {name}
-                </div>
-                <Latency latencyNs={latencyNs} />
-            </div>
-            <div className="flex flex-row items-center justify-between text-muted-foreground">
-                <div
-                    className={`
-                    flex flex-row gap-x-1 items-center
-                    border border-currentColor
-                    text-[10px] font-bold
-                    pl-1 pr-1 rounded-md px-1 leading-4
-                    w-fit h-fit
-                `}
-                >
-                    {displayKind}
-                </div>
+    return 'Unknown';
+};
 
-                <div className="col-span-1 truncate break-all text-[13px]">
-                    {moment(parseInt(startTimeUnixNano) / 1000000).format(
-                        'HH:mm:ss',
-                    )}
-                </div>
-            </div>
-        </div>
-    );
+// Helper function to format latency - simple text, no animation overhead
```

Comment:
Since we've already unified the number representation, should we use the unified interface?